internal/provider/incident_workflow_resource.go

```diff
@@ -83,23 +109,7 @@ func (r *IncidentWorkflowResource) Schema(ctx context.Context, req resource.Sche
 									"operation": schema.StringAttribute{
 										Required: true,
 									},
-									"param_bindings": schema.SetNestedAttribute{
```

Comment:
`SetNestedAttribute` is "a Set of Nested Attibutes", not "Set these Nested Attributes"

which completely explains the lack of ordering

Reply:
OH right this makes sense

Reply:
yea I was getting workflow validation errors because my params were in the wrong order